packages/cloudvision-grpc-web/types/resource.ts

```diff
@@ -5,8 +5,7 @@ import { grpc } from '@improbable-eng/grpc-web';
  * A GRPC response the represents a response from a streaming resource.
  */
 export interface StreamingResourceResponse extends grpc.ProtobufMessage {
-  getType?(): OperationMap[keyof OperationMap];
-  setType?(value: OperationMap[keyof OperationMap]): void;
+  type?: OperationMap[keyof OperationMap];
```

Comment:
this is ts-protoc specific right?

Reply:
Yes. ts-proto doesn't use get/set but instead has the type property in the response